estimation_stage_only.ipynb

```diff
@@ -114,46 +105,21 @@
+      "File \u001b[0;32m~/rail_deepdisc/src/rail/estimation/algos/deepdisc.py:528\u001b[0m, in \u001b[0;36mDeepDiscPDFEstimatorWithChunking.run\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    526\u001b[0m \u001b[38;5;66;03m# process this chunk of data\u001b[39;00m\n\u001b[1;32m    527\u001b[0m \u001b[38;5;28mprint\u001b[39m(\u001b[38;5;124mf\u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mProcessing chunk (start:end) - (\u001b[39m\u001b[38;5;132;01m{\u001b[39;00mstart_idx\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m:\u001b[39m\u001b[38;5;132;01m{\u001b[39;00mend_idx\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m)\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n\u001b[0;32m--> 528\u001b[0m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_process_chunk\u001b[49m\u001b[43m(\u001b[49m\u001b[43mstart_idx\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mmetadata\u001b[49m\u001b[43m)\u001b[49m\n",
+      "File \u001b[0;32m~/rail_deepdisc/src/rail/estimation/algos/deepdisc.py:542\u001b[0m, in \u001b[0;36mDeepDiscPDFEstimatorWithChunking._process_chunk\u001b[0;34m(self, start_idx, metadata)\u001b[0m\n\u001b[1;32m    530\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21m_process_chunk\u001b[39m(\u001b[38;5;28mself\u001b[39m, start_idx, metadata):\n\u001b[1;32m    531\u001b[0m \u001b[38;5;250m    \u001b[39m\u001b[38;5;124;03m\"\"\"For a given block of images and metadata, calculate the PDFs and\u001b[39;00m\n\u001b[1;32m    532\u001b[0m \u001b[38;5;124;03m    write them to a temporary file.\u001b[39;00m\n\u001b[1;32m    533\u001b[0m \n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m    539\u001b[0m \u001b[38;5;124;03m        The list of metadata dictionaries for this block of images\u001b[39;00m\n\u001b[1;32m    540\u001b[0m \u001b[38;5;124;03m    \"\"\"\u001b[39;00m\n\u001b[0;32m--> 542\u001b[0m     \u001b[38;5;28;01mwith\u001b[39;00m \u001b[43mmp\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mMananger\u001b[49m() \u001b[38;5;28;01mas\u001b[39;00m manager:\n\u001b[1;32m    543\u001b[0m         q \u001b[38;5;241m=\u001b[39m manager\u001b[38;5;241m.\u001b[39mQueue()\n\u001b[1;32m    545\u001b[0m         \u001b[38;5;66;03m# call detectron2's `launch` function to parallelize the inference\u001b[39;00m\n",
+      "\u001b[0;31mAttributeError\u001b[0m: module 'torch.multiprocessing' has no attribute 'Mananger'"
```

Comment:
Wondering if this is ok/was the output it replaced created after switching to use of mp.Manager in algos/deepdisc.py?

I've tried running this myself to see, but am getting a `version GLIBCXX_3.4.30' not found` from the import cell, so I'll take another look at it in the morning.